### PR TITLE
[Project 0] Dane Christensen - Setup Hello World Code with box

### PR DESCRIPTION
**Pull request by chr11031**
Just getting the dev environment setup so that I can get started with computer graphics!

Screenshot below:
snap

### CODE REVIEW

File: io.cpp [hunk 1]
```diff
@@ -80,11 +80,21 @@ int main()
...
 
+	// Draw a small box
+	for(int y = 50; y < 100; y++)
+	  {
+	    for(int x = 50; x < 100; x++)
+	      {
+		framePtr[y][x] = 0x0ffffffff;
+	      }
+	  }
+
```
Comment (danebear) [inline io.cpp:96]: Why did you choose 50x50? Why not something a bit bigger?

File: io.cpp [hunk 2]
```diff
@@ -80,11 +80,21 @@ int main()
         {
             for(int x = 0; x < S_WIDTH; x++)
             {
-                // Red color
-                framePtr[y][x] = 0x00ff0000;
+                // Black color
```
Comment (danebear) [inline io.cpp:83]: Why not something more interesting?

Comment (chr11031) [reply io.cpp:83]: Guess I'm colorblind.

File: io.cpp (hunk 1 above)
Comment (chr11031) [reply io.cpp:96]: I think because my monitor is a small resolution it seemed normal. Will make bigger for future projects though.

**Comment by danebear** [comment]
Looks good!

**Pull request merged**